occurrences/models.py

```diff
@@ -813,18 +813,21 @@ def send_enrolment_summary_report_to_providers(cls, days=1):
         # Send enrolment summary
         occurrences_services.send_enrolment_summary_report_to_providers(enrolments)
 
+    def get_contact_people(self) -> list[Person]:
```

Comment:
Although we're using Python 3.9 which supports list[Person] I'm not sure whether some other part in the CI/CD pipeline doesn't as it has [broken](https://dev.azure.com/City-of-Helsinki/kultus/_build/results?buildId=143290&view=logs&j=8267e077-3a31-5bba-93c5-4b3e55896ad0&t=45ce9a3b-1c36-5c65-31cd-c5d7e373a2ce) on this line:
```
 def get_contact_people(self) -> list[Person]:
TypeError: 'type' object is not subscriptable
```

Reply:
yea, dunno 😞 Changed it to `typing.List` now.